src/test/ui/consts/const-eval/assign-to-static-within-other-static-2.rs

```diff
@@ -1,7 +1,7 @@
 // New test for #53818: modifying static memory at compile-time is not allowed.
 // The test should never compile successfully
 
-#![feature(const_raw_ptr_deref)]
+#![feature(const_raw_ptr_deref, const_mut_refs)]
 
```

Comment:
This test was only failing pre-monomorphization due to a missing feature gate. It seemed like we wanted to exhaustively test this, especially since there is another test *without* the feature gate modified in this commit. This led to #77353.